src/AdminConsole/Program.cs

```diff
@@ -68,11 +72,26 @@ void RunTheApp()
         IConfigurationSection ddConfig = ctx.Configuration.GetSection("Datadog");
         if (ddConfig.Exists())
         {
+            var assembly = Assembly.GetExecutingAssembly();
+            var version = assembly.GetInformationalVersion() ??
+                              assembly.GetName().Version?.ToString() ??
+                              "unknown version"; // 1.2.3-ci-SHA
+
+            // TRY TO GET DataDog Azure App Service Extension to use a version configured from code
+            // instead of DD_VERSION environment variable
+            // Doesn't seem to work though? 
+
+            // Configure the DataDog Tracer with version
+            var settings = TracerSettings.FromDefaultSources();
+            settings.ServiceVersion = version;
+            Tracer.Configure(settings);
+
             var ddKey = ddConfig.GetValue<string>("ApiKey");
             if (!string.IsNullOrWhiteSpace(ddKey))
             {
                 config.WriteTo.DatadogLogs(
                     apiKey: ddKey,
+                    tags: new[] { "version:" + version },
                     configuration: new DatadogConfiguration(ddConfig.GetValue<string>("url")));
```

Comment:
It would be ideal if this were automatic, but the fact that even just configuring the version manually doesn't work is really weird 🤔